toml/prognostic_edmfx_bomex_explicit.toml

```diff
@@ -0,0 +1,62 @@
+[C_E]
+value = 0.0176
+
+[EDMF_surface_area]
+value = 0.1
+
+[EDMF_min_area]
+value = 1.0e-5
+
+[EDMF_max_area]
+value = 0.5
+
+[entr_inv_tau]
+value = 0.0002
+
+[entr_coeff]
+value = 0.3
+
+[min_area_limiter_scale]
+value = 0
+
+[min_area_limiter_power]
+value = 10
+
+[detr_inv_tau]
+value = 0
+
+[detr_coeff]
+value = 0
+
+[detr_buoy_coeff]
+value = 0
+
+[detr_vertdiv_coeff]
+value = 0
+
+[detr_massflux_vertdiv_coeff]
+value = 1
+
+[max_area_limiter_scale]
+value = 0
+
+[pressure_normalmode_drag_coeff]
+value = 20.0
```

Comment:
This value is different from Costa's namelist because of differences in the nonhydrostatic pressure drag (using scale height in climaatmos but updraft top in tc).